### DISCUSSION TITLE
Primi passi prime domande

### DISCUSSION THREAD

**Discussion by fabrua7** [indomus/forum#2474] [GENERALE - Discussioni generiche]
Buongiorno a tutti, sono nuovo in questo mondo, o meglio sto per entrarci (a parte qualche faretto ikea e termostato netatmo in mio possesso), vorrei farvi queste domande. Partendo con il presupposto che vorrei per prima cosa prendere un mini pc per installarci sopra HA e la chiavetta zigbee della sonoff. La mia domanda è questa qui avendo io delle luci comandabili accensione/spegnimento da più punti di casa per rendere queste luci domotiche posso installare un attuatore tipo sonoff o pulsantiera smart bticino living now su un unico pulsante? o sono obbligato a rendere smart ogni pulsante che comanda le luci? cosa mi suggerite per rendere smart le luci, i pulsant bticino (mi piacciono molto esteticamente) o un sonoff interno alla cassetta? grazie tante

**Comment by essegi65** [comment]
Dipende dal tuo budget, dalla tua "esperienza" a riguardo l'elettricità e se vuoi modificare o meno l'impianto elettrico attuale.
Ovvero la soluzione che ti permette di NON modificare l'impianto elettrico e quindi non chiamare l'elettricista e/o certificare di nuovo l'impianto e non modificare niente rispetto all'esistente sarebbe la soluzione con gli interruttori BTICINO serie living now (anche altri produttori hanno i flutti connessi però) purtroppo è anche la soluzione più costosa. Se vuoi inserire dei device connessi (sonoff, shelly, ecc.) sono senz'altro più economici però comportano una modifica all'impianto elettrico esistente. Personalmente ho scelto sia l'una che l'altra dove c'erano già deviatori e/o interruttori ho messo la serie living, per portare nuovi punti luce ho adottato shelly e/o sonoff

**Reply by fabrua7** [reply]
diciamo che non sono elettricista, non sono uno che si diletta al fai da te ma quando mi ci metto, cerco di capire come funziona con tutorial, video o altro e lo faccio, nel senso non mi sarebbe difficile inserire in una 503 un sonoff o uno shelly. devo solo capire quanti mi ci vorrebbero. per esempio le luci dell'ingresso mi si accendono dall'ingresso di casa e dal corridoio. dovrei mettere un sonoff/shelly ? se si dove? con un sonoff/shelly si può comandare la luce anche fisicamente? es.. ho i faretti ikea se il pulsante fisico sta su on stanno collegati e posso spegnere e accendere con alexa, se invece sta su off i faretti risultano disconessi e quindi si possono accendere solo manualmente, quindi per farli funzionare vocalmente tengo sempre il pulsante su on, spero di essere stato chiaro. ma i sonoff/shelly non sono zigbee? bticino si? anche perchè la risposta potrebbe influenzare la mia scelta , perchè mettere 10 comandi wifi solo per le luci per tutta casa è sicuramente più energivoro e mi intaserrebero di più la rete rispetto a 10 comandi zigbee. o sbaglio? scusa per il post lunghissimo

**Reply by essegi65** [reply]
Allora per riepilogare: devi mettere un sonoff/shelly per ogni luce/i ovvero per le tue luci d'ingresso occorre un sonoff/shelly che devi inserire in una scatola 503 o di derivazione (attenzione per questo dico devi avere manualità e sapere cosa stai facendo la corrente è pericolosa) e poi portarci i fili del deviatore secondo lo schema che trovi sia su sonoff che su shelly. Se metti dei frutti connessi (tipo Bticino, vimar, ecc.) tutto questo non lo devi fare si tratta solo di sostituire il vecchio con il nuovo. Una volta collegati secondo lo schema adeguato li puoi comandare da pulsante/interruttore (frutto della 503) o tramite HA o assistenti vocali. Gli shelly al momento sono solo wifi mentre i sonoff li trovi sia wifi che zigbee (p.s. a livello di consumo cambia poco, a livello di occupazione wifi cambia)
